### PR TITLE
add instructions for debug setup

### PR DESCRIPTION
**Pull request by yangwwei**
this will be useful for developers to debug issues locally

### CODE REVIEW

**Comment by yangwwei** [comment]
@wilfred-s could you please help to review this?

File: docs/setup/env-setup.md [hunk 1]
```diff
@@ -87,6 +87,24 @@ Without setting the enviromnent minikube might not find the docker images when d
...
 
+## Debug code locally
+
+Note, this instruction requires you have GoLand IDE for development.
+
+In GoLand, go to yunikorn-k8shim project. Then click "Run" -> "Debug..." -> "Edit Configuration..." to get the pop-up configuration window,
+
+![Debug Configuration](../images/goland_debug.jpg)
+
+The highlighted fields are the configurations you need to add. These include:
```
Comment (wilfred-s) [inline docs/setup/env-setup.md]: The option `Go Build` per the left hand side is not open/added by default
might need to explain that

File: docs/setup/env-setup.md [hunk 2]
```diff
@@ -87,6 +87,24 @@ Without setting the enviromnent minikube might not find the docker images when d
...
+
+- Run Kind: package
+- Package path: point to the path of `pkg/shim` package
+- Working directory: point to the path of the `conf` directory, this is where the program loads configuration file from
+- Program arguments: specify the arguments to run the program, such as `-kubeConfig=/path/to/.kube/config -interval=1s -clusterId=mycluster -clusterVersion=0.1 -name=yunikorn -policyGroup=queues -logEncoding=console -logLevel=-1`.
```
Comment (wilfred-s) [inline docs/setup/env-setup.md]: This text hides the `/path/to/.kube` reference and I did not see it until I copied it in and walked over each parameter to see what it did.

File: docs/setup/env-setup.md [hunk 3]
```diff
@@ -87,6 +87,24 @@ Without setting the enviromnent minikube might not find the docker images when d
 1. run the image build from the yunikorn-k8shim repository root: `make image`
 1. deploy the scheduler as per the normal instructions.
 
+## Debug code locally
+
+Note, this instruction requires you have GoLand IDE for development.
+
+In GoLand, go to yunikorn-k8shim project. Then click "Run" -> "Debug..." -> "Edit Configuration..." to get the pop-up configuration window,
```
Comment (wilfred-s) [inline docs/setup/env-setup.md]: The `Go Build` option is not visible by default and you might need to add it via the + sign if it is not there

**Comment by yangwwei** [comment]
Updated per @wilfred-s 's comment, please help to review again. Thx

**Pull request merged**